lblod/frontend-gelinkt-notuleren
# GN-5207: meeting attendee modal - only show mandatees with a linked person

### Overview
This PR ensures that only mandatees with a linked 'is-bestuurlijke-alias-van' person are displayed in the meeting attendee modal.

##### connected issues and PRs:
[GN-5207](https://binnenland.atlassian.net/browse/GN-5207)

### Setup
None

### How to test/reproduce
- Start the app
- Login as a municipality which has the following conditions:
   * Legislation is already being prepared in LMB
   * But: not all mandatees are filled in yet
- Create/open an IV
- When editing the attendees of the meeting/an AP, ensure that only mandatees linked to a person are displayed (a.k.a. the firstname/lastname columns are filled in)


### Challenges/uncertainties
- I did not add any error handling on the 'download notulen' action
- We should really add a loading state to the attendee modal, but will do so in a seperate PR.



### Checks PR readiness
